.travis/test_04_install.sh

```diff
@@ -24,3 +27,12 @@ fi
 travis_retry DOCKER_EXEC apt-get update
 travis_retry DOCKER_EXEC apt-get install --no-install-recommends --no-upgrade -qq $PACKAGES $DOCKER_PACKAGES
 
+# macOS build - does not use docker, but might require some packages
+elif [ "$TRAVIS_OS_NAME" = "osx" ]; then
+
+for package in $PACKAGES; do
+  brew install $package || true # brew fails if the package is already available
```

Comment:
```
brew install $package || true # brew fails if the package is already available
             ^-- SC2086: Double quote to prevent globbing and word splitting.
```